ResearchKit/Common/ORKImageCaptureStep.h

```diff
@@ -65,4 +65,12 @@ ORK_CLASS_AVAILABLE
  */
 @property (nonatomic) UIEdgeInsets templateImageInsets;
 
+/**
+ Accessibility hint to be used in Voice Over for `templateImage`.
+ 
+ The accessibility hint provides voice over instructions for the user. When setting the
+ accessibility hint, be sure to clearly state what the 'templateImage' is displaying.
+ */
+@property (nonatomic, strong) NSString *templateImageAccessibilityHint;
```

Comment:
A template image is NOT required in order to use the ``ORKImageCaptureStep``.  Since the existing accessibility hint is applied to the preview view, I do not believe this should be called ``templateImageAccessibilityHint``.  If you did not supply a template image, you may wish to provide a hint like "Take a picture of your hand" without mentioning alignment.